include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -9088,6 +9333,34 @@
         }
       ]
     },
+    {
+      "category" : "ValueEnum",
+      "kind" : "FPDenormMode",
+      "enumerants" : [
+        {
+          "enumerant" : "Preserve",
+          "value" : 0
```

Comment:
I think it is preferable to add:

     "capabilities" : [ "FunctionFloatControlINTEL" ],
     "version" : "None"

Reply:
Thanks! It wasn't mentioned in the [spec](https://github.com/intel/llvm/blob/39fa9b0cbfbae88327118990a05c5b387b56d2ef/sycl/doc/extensions/SPIRV/SPV_INTEL_float_controls2.asciidoc), but I guess you are right, I'll open a spec issue.